pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.en-gb.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "How to connect an OVHcloud domain name to a Google Site"
+excerpt: "Prepare and configure your OVHcloud domain name’s DNS zone to connect to a Google Site"
+updated: 2024-09-25
+---
+
+## Objective
+
+You have a domain name with OVHcloud and you want to connect it to a Google Site. In this guide, you will find the steps for preparing and configuring your OVHcloud DNS zone, so that you can configure your Google Site.
+
+**Find out how to connect your OVHcloud domain name to a Google Site.**
+
+> [!warning]
+>
+> - The Google Site support team does not have access to your OVHcloud domain name settings, and cannot advise you on the information you will need to provide them.
+> - OVHcloud provides services that you are responsible for configuring, managing and managing. It is therefore up to you to ensure that it works properly.<br><br> We have provided you with this guide in order to help you with common tasks as much as possible. Nevertheless, we recommend contacting a [specialist provider](/links/partner) and/or the service’s publisher if you encounter any difficulties. We will not be able to assist you. You can find more information in the [Go further](#gofurther) section of this guide.
+>
+
+## Requirements
+
+- You must be logged in to your [OVHcloud Control Panel](/links/manager).
+- A [domain name](/links/web/domains) registered with OVHcloud.
+- You must have the [appropriate permissions](/pages/account_and_service_management/account_information/managing_contacts) to manage the domain name from your [OVHcloud Control Panel](/links/manager).
+- You must own and own a Google Site.
+
+## In practice
+
+Before following the steps in this guide, we recommend that you get familiar with configuring a DNS zone by using our guide on [Editing an OVHcloud DNS zone](/pages/web_cloud/domains/dns_zone_edit).
+
+> [!warning]
+>
+> Your DNS zone is potentially already preconfigured or linked to a web hosting plan. We will look at how to identify each DNS record required to log in to your Google Site. Some records will need to be deleted to avoid conflicts with DNS records required in this configuration. Others will just have to be modified or created. For a better understanding, we will use the domain name **mydomain.ovh** as an example. Replace it with your domain name during configuration.
+
+### Step 1 - Configure your Google Site
+
+> [!warning]
+>
+> Only the owner of a Google Site can connect it to a domain name. If necessary, find out how to [change the owner of the Google site](https://support.google.com/sites/answer/97934).
```

Comment:
The link redirect to a FR page. Please could you change the FR link by the EN link ?

Reply:
Really ? For me it redirects to the official english page (the english language is picked up in the bottom of the page in the list). Are you sure your browser is not forcing the french translation ?)

Reply:
I agree with Fabien. My browser language is set to English, the link leads to a page in English.